### PR TITLE
Add mothership chart

### PR DESCRIPTION
**Pull request by eseiker**
Add charts/mothership and replace existing 9c-dev/l2launcher with it

supersedes https://github.com/planetarium/mothership-l2launcher/pull/34

TODO: change targetRevision into main before merge and create new cluster

! PR TO BE SQUASHED

### CODE REVIEW

File: 9c-dev/l2launcher-20240214/application.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: l2launcher-20240214
+  namespace: argocd
+spec:
+  project: default
+  sources:
+    - repoURL: https://github.com/planetarium/9c-infra.git
+      targetRevision: l2launcher
```
[the remark this answers is not in the document]
Comment (eseiker) [reply 9c-dev/l2launcher-20240214/application.yaml:10]: yes, should be changed to main before merge

File: charts/mothership/templates/env.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-cm
+  namespace: {{ .Release.Namespace }}
+data:
+  L1_RPC: {{ .Values.env.L1_RPC }}
+  L1_BEACON_RPC: "{{ .Values.env.L1_BEACON_RPC }}"
```
Comment (sky1045) [inline charts/mothership/templates/env.yaml:8]: if these are only used with env vars, we can inject directly, not through configmap like below
```
env:
  - name: L1_RPC
    value: "{{ .Values.env.L1_RPC }}"
```
or is there another reason we should use configmap?

Comment (eseiker) [reply charts/mothership/templates/env.yaml:8]: yup there's no reason to keep that configmap by now. it was legacy from manual pre-helm yamls xP

File: charts/mothership/templates/op-batcher.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                --num-confirmations=1 \
+                --safe-abort-nonce-too-low-count=3 \
+                --resubmission-timeout=30s \
+                --max-channel-duration=1
+          env:
+            - name: L1_RPC
+              valueFrom:
+                configMapKeyRef:
+                  name: env-cm
+                  key: L1_RPC
```
Comment (sky1045) [inline charts/mothership/templates/op-batcher.yaml:43]: need to inject `{{ $.Values.env.L1_RPC }}` as we removed configmap

**Review by sky1045 — APPROVED** [review]

**Pull request merged**